backend/models/StudentEvent.js

```diff
@@ -0,0 +1,45 @@
+import { Model } from 'sequelize';
+
+export default (sequelize, DataTypes) => {
+  class StudentEvent extends Model {}
+
+  StudentEvent.init(
+    {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataTypes.INTEGER,
+      },
+      student_id: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      meeting_id: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      event_action: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
```

Comment:
Can you add validation around this column so that the values can only be `JOINED` and `LEFT`? You can declare those as constants at the top of this file

You can see some documentation around that here: https://sequelize.org/docs/v6/core-concepts/validations-and-constraints/#validators